controllers/machineset_controller.go

```diff
@@ -519,6 +519,7 @@ func (r *MachineSetReconciler) waitForMachineDeletion(ctx context.Context, machi
 // MachineToMachineSets is a handler.ToRequestsFunc to be used to enqeue requests for reconciliation
 // for MachineSets that might adopt an orphaned Machine.
 func (r *MachineSetReconciler) MachineToMachineSets(o client.Object) []ctrl.Request {
+	log := ctrl.LoggerFrom(context.TODO(), "object", client.ObjectKeyFromObject(o))
```

Comment:
FYI, this will never log unless the global logger is set

Reply:
Added a comment